vendor/bat-native-ledger/src/bat/ledger/internal/database/database_event_log.cc

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <utility>
+
+#include "base/guid.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/common/time_util.h"
+#include "bat/ledger/internal/database/database_event_log.h"
+#include "bat/ledger/internal/database/database_util.h"
+#include "bat/ledger/internal/ledger_impl.h"
+
+using std::placeholders::_1;
+
+namespace braveledger_database {
+
+namespace {
+
+const char kTableName[] = "event_log";
+
+}  // namespace
+
+DatabaseEventLog::DatabaseEventLog(
+    bat_ledger::LedgerImpl* ledger) :
+    DatabaseTable(ledger) {
+}
+
+DatabaseEventLog::~DatabaseEventLog() = default;
+
+void DatabaseEventLog::Insert(
+    const std::string& key,
+    const std::string& value) {
+  if (key.empty() || value.empty()) {
+    BLOG_IF(1, key.empty(), "Key is empty");
+    BLOG_IF(1, value.empty(), "Value is empty");
+    return;
+  }
+
+  auto transaction = ledger::DBTransaction::New();
+
+  const std::string query = base::StringPrintf(
+      "INSERT INTO %s (event_log_id, key, value, created_at) "
+      "VALUES (?, ?, ?, ?)",
+      kTableName);
+
+  auto command = ledger::DBCommand::New();
+  command->type = ledger::DBCommand::Type::RUN;
+  command->command = query;
+
+  BindString(command.get(), 0, base::GenerateGUID());
```

Comment:
Do we need a GUID field? Will the row ever be correlated with any other data?

Reply:
currently no, but could be